components/scrapd-grid/component.js

```diff
@@ -36,24 +36,24 @@ class ScrapdGrid extends React.Component {
       },
       {
         title: 'Crash',
-        dataIndex: 'Fatal crashes this year',
+        dataIndex: 'crash',
         key: 'Fatal crashes this year',
         sortDirections: ['descend'],
         sorter: (a, b) => a['Fatal crashes this year'] - b['Fatal crashes this year']
```

Comment:
You must update the sort key, otherwise sorting will not work anymore.

Reply:
Completed.